plugins/outputs/prometheus_client/README.md

```diff
@@ -16,7 +16,7 @@ metrics on `/metrics` (default) to be polled by a Prometheus server.
   ## both plugins to ensure metrics are round-tripped without modification.
   ##
   ##   example: metric_version = 1;
-  ##            metric_version = 2; recommended version
+  ##            metric_version = 2;
```

Comment:
do we need a similar section about the differences?

Reply:
I added a pointer here too